### PR TITLE
fix: Use MaterialUI SvgIcons instead of Font

### LINKED ISSUE #6062: Use MaterialUI SvgIcons instead of Font
## Issue - Enhancement

Today, we ship two quite large font icons for Material UI:

```console
> ll -h packages/client/styles/theme/fonts/MaterialIcons*
-rw-r--r-- 1 blimmer staff 103K Feb  1 09:49 packages/client/styles/theme/fonts/MaterialIcons-Regular.woff2
-rw-r--r-- 1 blimmer staff 113K Feb  1 09:49 packages/client/styles/theme/fonts/MaterialIconsOutlined-Regular.woff2
```

This pattern means that we ship every icon (used or unused) to the client, which is inefficient. If we switched over to the React `@mui/icons-material` package, we could use the SVG versions of just the icons we use in the app. This should result in a far smaller payload to deliver the icons we use.

Additionally, on slow internet connections, there's a strange behavior where you see the "name" of the icon in the DOM while the font is loading. This breaks the layout and overall looks a bit odd. Check out this Loom recording to see what I'm talking about: https://www.loom.com/share/9bde6426362d4c8ca97ff7a3d6039a83

## Accessibility

I noticed this issue while working on the follow-up to #5961. When I requested the text content of a submitted retrospective item, the string I entered oddly had `"cancelsearch"` appended to the end of string:

```
Error: expect(received).toHaveText(expected)

Expected string: "We should start doing this!"
Received string: "We should start doing this!cancelsearch"
Call log:
  - expect.toHaveText with timeout 5000ms
  - waiting for selector "[data-cy=reflection-column-Start] [data-cy=reflection-stack-Start]"
```

When I inspected the HTML element, indeed, the `<i>` element has the name of the element:

Screen Shot 2022-02-15 at 12 59 18

I listened to this with VoiceOver on my Mac and it wasn't announcing the `cancel` text, so I don't think there's necessarily an accessibility issue with the current pattern. However, some screen readers might also behave differently than VoiceOver.

### Acceptance Criteria

- SVG material icons are used instead of the MUI font icons.
- Unused icons aren't included in the final bundle (see https://mui.com/components/icons/#usage).

**Estimated effort:** 12 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/parabol/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by Dschoordsch**
# Description

Fixes #6062

Use Material UI SVG icons instead of the font

## Demo

No demo, app should look the same.

## Testing scenarios

Use the whole app, ideally running release testing procedure to check that all screens look as intended.

## Final checklist

- [ ] I checked the [code review guidelines](../docs/codeReview.md)
- [ ] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [ ] I have performed a self-review of my code, the same way I'd do it for any other team member
- [ ] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [ ] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [ ] PR title is human readable and could be used in changelog

### CODE REVIEW

**Comment by Dschoordsch** [comment]
The current implementation increases the bundle size. Since the images are components, these are also not cached, degrading the performance overall.

**Comment by 87prashant** [comment]
> degrading the performance overall

That's sad, any way we can improve it?

**Comment by Dschoordsch** [comment]
> > degrading the performance overall
> 
> That's sad, any way we can improve it?

I think we have to go the SVG route, i.e. downloading the used icons as SVG and using those as images. This will probably not change the download size much, but it makes it possible to cache the images in the browser. As components these will need to be loaded each release. Not a huge deal, but when we touch this, we should do it right.

**Comment by Dschoordsch** [comment]
> > > degrading the performance overall
> > 
> > 
> > That's sad, any way we can improve it?
> 
> I think we have to go the SVG route, i.e. downloading the used icons as SVG and using those as images. This will probably not change the download size much, but it makes it possible to cache the images in the browser. As components these will need to be loaded each release. Not a huge deal, but when we touch this, we should do it right.

I made a mistake when measuring as I did not properly minimize the chunks (I did not pass `--deploy`). When properly done, the size is virtually identical, so we can move on with this.

**Comment by Dschoordsch** [comment]
@igorlesnenko I know you've taken a look at this before the rebase, but could you please just do a quick glance over if there is something obviously wrong?

**Review by 87prashant — CHANGES_REQUESTED** [review]

**Review by igorlesnenko — CHANGES_REQUESTED** [review]
What has caught my eye:

Checkbox is now too close:

image

Different checkbox sizes:

image

Next button is a half-pixel bolder than it was. Maybe not critical.

image

Everything else looks good

File: packages/client/components/BottomControlBarReady.tsx [hunk 1]
```diff
@@ -31,17 +31,21 @@ interface Props {
...
     height: 24,
+    width: 24,
     opacity: isNext ? 1 : isViewerReady ? 1 : 0.5,
     transformOrigin: '0 0',
     // 20px to 16 = 0.75
     transform: progress > 0 ? `scale(0.75)translate(4px, 4px)` : undefined,
-    transition: `transform 100ms ${BezierCurve.DECELERATE}`
+    transition: `transform 100ms ${BezierCurve.DECELERATE}`,
+    '& svg': {
+      // without fill property the icon will have different color inside
```
[the remark this answers is not in the document]
Comment (Dschoordsch) [reply packages/client/components/BottomControlBarReady.tsx:44]: updated it

**Review by igorlesnenko — APPROVED** [review]
LGTM!

**Pull request merged**